docs/decision_record.md

```diff
@@ -6,6 +6,22 @@
 
 [TOC]
 
+## 2023-01-11 - Array names should, but are not required to be ASCII
+
+### Decision
+
+We RECOMMEND that custom array names be ascii characters. We expect most implementations will require this. Nevertheless, implementers may choose to allow users to use UTF-8 characters as an extension to the spec. Implementing UTF-8 will not be required for an implementation. In this extension, array names MUST at least follow UTF-8.
```

Comment:
Suggest `RECOMMEND` change to `SHOULD` to remain inline with https://www.rfc-editor.org/rfc/rfc2119